docs/Automation.md

```diff
@@ -0,0 +1,265 @@
+## Axe.Windows - Automation
+
+### Overview
+To enable various automation scenarios, we have created an assembly
+(`Axe.Windows.Automation.dll`) that exposes a subset of core
+AxeWindows functionality to automation systems. This can be used in
+a variety of ways, including as part of a standalone test system, as part of a
```

Comment:
nit: 'as part of a' is used redundantly here and can be replaced this with:

'including as part of a standalone test system, PowerShell script, or CI/CD solution'